test/e2e/setup_test.go

```diff
@@ -186,6 +186,15 @@ var (
 		ignoreConditionLTTAndMessageFields,
 		cmpopts.EquateEmpty(),
 	}
+
+	safeDeploymentCRPStatusCmpOptions = cmp.Options{
+		cmpopts.SortSlices(lessFuncCondition),
+		cmpopts.SortSlices(lessFuncResourceIdentifier),
+		cmpopts.SortSlices(lessFuncFailedResourcePlacements),
```

Comment:
I wonder how the compare works when we don't know which cluster gets the bad deplolyment in this case?

Reply:
Since we don't sort placementStatus using clusterName, we can instead sort using conditions length since there should be only one placementStatus with available condition, other placementstatus should have rollout stopped condition